docs/source/using/profiles.adoc

```diff
@@ -43,16 +61,17 @@ $ minishift profile list
 [[switching-profiles]]
 == Switching Profiles
 
-With the help of xref:../command-ref/minishift_profile_set.adoc#[`minishift profile set`] you can switch between profiles or set a profile as active.
-When a profile is active, it is not required to pass `--profile` flag to run minishift commands against that profile.
+With the help of xref:../command-ref/minishift_profile_set.adoc#[`minishift profile set`] you can switch between profiles.
```

Comment:
I would rewrite this sentence so that we immediately tell the user _what_ they can do followed by _how_ they can do it:

"You can switch between profiles with the xref:../command-ref/minishift_profile_set.adoc#[`minishift profile set`] command."